README.md

```diff
@@ -1 +1,65 @@
-# The Odin Project Ruby Exercises
\ No newline at end of file
+# The Odin Project Ruby Exercises
+
+# About this repository
+
+This repository is a complement to the Basic Ruby course for The Odin Project. It contains a series of basic Ruby exercises alongside pre-written tests that will help guide the student towardsa solution, providing a Test Driven Development experience.
+
+The exercises are meant to be done in the order dictated by their numbers (1, 2, 3...). You can still do them out of order, provided you feel appropiately prepared for them, however using the intended order would be ideal.
+
+# Cloning and installation
+
+Clone this repository to your local machine. To do this, click on the green "Code" button at the top right of list of files and copy the HTTPS link in the prompt. Then, run the following command in your terminal:
+
+    git clone <link>
+
+This will create a folder called 'ruby-exercises' inside the directory you ran the command in.
+
+CD into that folder:
+
+    cd ruby-exercises
+
+Then run the following command:
+
+    bundle install
+
+This folder contains a Gemfile, which tells bundle which gems to install locally and makes them available for use in that directory. In this case, we are installing RSpec, which is a popular Ruby testing framework.
+
+Verify that the installation was succesful by simply running the following command:
+
+    rspec
+
+Which should print a few lines about not finding any examples. This is ok and means it's working!
+
+# Usage
```

Comment:
Do you think we should move this usage section to the README in the Ruby Basics folder? or maybe have there as well?

Reply:
I think I would leave it as is, and maybe even delete the README in the Ruby Basics folder. I think cloning it is overkill, so it would be a choice between that section to the Ruby Basics README or deleting that README.

Reply:
They way I am envisioning this working is we will link to the ruby_basics folder from ruby basics lessons. I think we will need some documentation there about how to run the tests as it will probably be the users first experience with running tests. But we can leave that to later :)